src/blockchain/clients/blockchain_client.ts

```diff
@@ -12,4 +12,6 @@ export interface BlockchainClient {
   getBoxesByTokenId<T extends AmountType = string>(
     tokenId: TokenId,
   ): AsyncGenerator<Box<T>[]>;
+
+  getMempool(): Promise<SignedTransaction[]>;
```

Comment:
@arobsn do you have any feelings on adding the methods needed by the monitoring component to the blockchain client used by plugins?

I was on the fence on if it should be separate so plugin devs are guided into implementing the `on*` methods instead of calling the APIs manually - decided on it probably not being a big deal for now

Reply:
I don't think that exposing such a heavy endpoint is something good, on the other side, bots must be able to check the the mempool. 

What do you think about holding the last mempool snapshot as an in-memory store? This way we avoid new calls if plugins need it the the mempool.

Reply:
Yep this is a good idea, will do this